OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -371,4 +372,4 @@
   </Target>
   -->
   <ItemGroup />
-</Project>
+</Project>
```

Comment:
There was a pull request that explicitly added those so I think you should also undo them before pushing.